### ISSUE TITLE
Postgres id not marked as `Generated` field

### ISSUE THREAD

**Issue by Joshuabaker2** [valtyr/prisma-kysely#1]
Hey, first of all, thanks for creating this package! Super awesome, exactly what I was looking for. I tried it out on my prisma schema, but the types don't come out quite right for what I'm expecting. Here's a simple example:

```prisma
model Transaction {
  id        String   @id @default(uuid())
  createdAt DateTime @default(now())

  foo String?
}
```

Generates:

```ts
export type Transaction = {
  id: string;
  createdAt: Generated<Timestamp>;
  foo: string | null;
};
```

Whereas I'd expect the actual result to be:
```ts
export type Transaction = {
  id: Generated<string>; // id should be marked as generated
  createdAt: Generated<Timestamp>;
  foo: string | null;
};
```

**Comment by Joshuabaker2** [comment]
Oh, I see now, uuid is done in the JS level (thanks for the comment in the source code). Since I'm using Postgres, I can change the model to use postgres' `dbgenerated` instead, like: 

```prisma
model Transaction {
  id        String   @id @default(dbgenerated("gen_random_uuid()"))
  createdAt DateTime @default(now())

  foo String?
}
```

Which marks it as `Generated`.

**Comment by valtyr** [comment]
@Joshuabaker2 Yes exactly! I should probably mention this in the README. I really appreciate the issue though, let me know if there's anything else that's weird that you come across.

**Comment by jokull** [comment]
It would be great to mention this in the README because users of this library probably want to opt out of all the js-runtime stuff Prisma does to columns.

**Comment by valtyr** [comment]
@jokull I've added a section to the README explaining this 🤝
